packages/vue-router/src/router.ts

```diff
@@ -62,6 +66,13 @@ export const createIonRouter = (opts: IonicVueRouterOptions, router: Router) =>
   const initialHistoryPosition = opts.history.state.position as number;
   let currentHistoryPosition = opts.history.state.position as number;
 
+  const currentHistoryIndex = () => {
```

Comment:
We should be able to inline this:
```
const currentHistoryIndex = () => currentHistoryPosition - initialHistoryPosition;
```

Reply:
The only thing I would say about this is debugging isn't possible. 

Short hand is fun and all for rapidly writing code but if you ever need to go back and debug, then you have to refactor it.